internetarchive/cli/ia_simplelists.py

```diff
@@ -0,0 +1,145 @@
+"""
+ia_simplelists.py
+
+'ia' subcommand for managing simplelists on archive.org.
+"""
+
+# Copyright (C) 2012-2025 Internet Archive
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import annotations
+
+import argparse
+import json
```

Comment:
```suggestion
from internetarchive.utils import json
```

to make use of ultrajson when installed, like elsewhere in the code.

Reply:
I tried to use the "Commit suggestion" feature here (new to me) and it made the commit under my main Github profile (probably will do the same for this comment). So, anyway, I didn't want it that way. And it was failing linter checks, so I reverted the change.